.pre-commit-config.yaml

```diff
@@ -14,7 +14,6 @@ repos:
     -   id: check-json
     -   id: pretty-format-json
         args: ['--no-sort-keys']
-    -   id: check-symlinks
```

Comment:
but we really shouldn't be putting broken symlinks in even for tests I would think.  Or maybe set an exclusion instead?

Reply:
IIUC, it's complaining that there's a [perfectly valid] relative symlink.

Reply:
Let's see...
```
check for broken symlinks................................................Failed
- hook id: check-symlinks
- exit code: 1

libfwupdplugin/tests/sys/devices/pci0000:00/0000:00:14.0/usb1/1-1/1-1:1.1/driver: Broken symlink
```

```
~/src/fwupd/libfwupdplugin/tests/sys/devices/pci0000:00/0000:00:14.0/usb1/1-1/1-1:1.1$ ls -alh
total 36K
drwxr-xr-x 3 supermario supermario 4.0K Jun 27 14:18 .
drwxr-xr-x 3 supermario supermario 4.0K Jun 27 14:18 ..
drwxr-xr-x 3 supermario supermario 4.0K Jun 27 14:18 0003:093A:2862.0076
-rw-r--r-- 1 supermario supermario    3 Jun 27 14:18 bInterfaceClass
-rw-r--r-- 1 supermario supermario    3 Jun 27 14:18 bInterfaceNumber
-rw-r--r-- 1 supermario supermario    3 Jun 27 14:18 bInterfaceProtocol
-rw-r--r-- 1 supermario supermario    3 Jun 27 14:18 bInterfaceSubClass
lrwxrwxrwx 1 supermario supermario   34 Jun 27 14:18 driver -> ../../../../../bus/usb/drivers/usb
-rw-r--r-- 1 supermario supermario   50 Jun 27 14:18 modalias
lrwxrwxrwx 1 supermario supermario   22 Jun 27 14:18 subsystem -> ../../../../../bus/usb
-rw-r--r-- 1 supermario supermario  141 Jun 27 14:18 uevent
```

```
$ cd ../../../../../
(fwupd) supermario@AUS-P9-MLIMONCI:~/src/fwupd/libfwupdplugin/tests/sys/devices$ ls
pci0000:00
```

I would say that's an improperly crafted test directory.  I don't see a `bus/usb` folder

Reply:
>  I don't see a bus/usb folder

Aha, git doesn't add empty dirs. Fixed I think.

Reply:
Looks like one more:

libfwupdplugin/tests/sys/devices/pci0000:00/0000:00:14.0/usb1/1-1/1-1:1.1/0003:093A:2862.0076/hidraw/hidraw1/subsystem: Broken symlink

Reply:
Done.